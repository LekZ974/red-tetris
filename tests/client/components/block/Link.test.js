import React from 'react';
import Link from '../../../../src/client/components/block/Link';
import renderer from 'react-test-renderer';

test('Link is display', () => {
  const component = renderer.create(
    <Link page="http://www.facebook.com">Facebook</Link>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
