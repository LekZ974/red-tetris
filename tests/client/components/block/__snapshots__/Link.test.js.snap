// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link is display 1`] = `
<a
  className="css-e3676l"
  color="blue"
>
  Facebook
</a>
`;
